File: src/main/java/de/maggiwuerze/xdccloader/model/transport/ServerTO.java
package de.maggiwuerze.xdccloader.model.transport;

import de.maggiwuerze.xdccloader.model.entity.Channel;
import de.maggiwuerze.xdccloader.model.entity.Server;
import java.time.LocalDateTime;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Value;

@Value
public class ServerTO {

	Long id;

	String name;

	String serverUrl;

	public ServerTO (Server server) {
		this.id = server.getId();
		this.name = server.getName();
		this.serverUrl = server.getServerUrl();
	}

}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/model/transport/ServerTO.kt
package de.maggiwuerze.xdccwebloader.model.transport

import de.maggiwuerze.xdccwebloader.model.entity.Server
import java.util.*

class ServerTO(server: Server) {
    var id: UUID? = server.id

    var name: String = server.name

    var serverUrl: String = server.serverUrl
}
